pkg/admissioncontroller/webhooks/admission/seedrestriction/admission.go

```diff
@@ -179,7 +179,7 @@ func (h *handler) admitBackupEntry(ctx context.Context, seedName string, request
 		return admission.Errored(http.StatusBadRequest, err)
 	}
 
-	if resp := h.admit(seedName, backupEntry.Spec.SeedName); !resp.Allowed {
+	if resp := h.admit(seedName, backupEntry.Spec.SeedName, backupEntry.Status.SeedName); !resp.Allowed {
```

Comment:
I only see `update` calls for this use-case in the migration flow after having a quick look. Can you elaborate why we need to cover `BackupEntries` as well?

Reply:
Indeed, it seems to not be needed. Reverted it.
/squash